### ISSUE TITLE
Install notes suggestions

### ISSUE THREAD

**Issue by carehart** [misterdai/cftracker#17]
So far so good. Just a couple of thoughts on the installation notes (install.txt): 
- i think some may miss the step to do the addition of the scheduled task, since it's buried within the steps one and 2. I wonder if it may be better to split it out to its own step (making the current 1 and 2 a I.1 and I.2, then the schedule task step a II.1). I do see that you have a reminder to them after step 2, so that just kind of reinforces my point above.
- in fact, why force them to create the scheduled manually when you could offer a template that uses CFSCHEDULE to do it for them? Indeed, you could have a step that's run during first execution of the app, that uses CFSCHEDULE first to check if it's been added, and if not, just adds it.
- also, since I'd think most people don't bother with extending their CF Admin interface, while that feature is nice and it's great that you show how to do it, I'd suggest you may want to put that second and the standalone first.
- in fact, if someone does the standalone form, couldn't they still do the Admin extension?
- that makes me wonder why the you don't just t make the installation three steps: drop in the code, add the scheduled task (if you need to keep it manual for some reason), then (optionally) extend the CF Admin.

Keep up the good work.
